docs/source/backends.rst

```diff
@@ -27,20 +27,13 @@ Generally speaking these backends also need to handle their own execution.
 They work by translating each node into a string, and passing the generated
 string to the database through a driver API.
 
-Impala
-******
-
-TODO
-
-Clickhouse
-**********
-
-TODO
-
-BigQuery
-********
-
-TODO
+- Impala
+- Clickhouse
+- BigQuery
+- HDFS
```

Comment:
can you add links to the actual docs for the backends

Reply:
sounds good! I am doing that. thanks!